### DISCUSSION TITLE
Pandoc and latex Installation

### DISCUSSION THREAD

**Discussion by d4r3-w0lf** [Cyfrin/security-and-auditing-full-course-s23#256] [General]
Can anyone help me with installation of pandoc and latex?

step by step guide.

**Comment by EngrPips** [comment]
Hello @Nandeesh-D, Please check the written lesson for the Lesson. It has all the guides to help you install pandoc and Latex and also includes some debugging tips in case you run into issues.

**Comment by d4r3-w0lf** [comment]
Thanks for the reply , but I installed it by following the written article
of the lesson on cyfrin updraft

**Reply by EngrPips** [reply]
Awesome, I am glad that did help you.
